otherlibs/dune-site/test/site-debug.t

```diff
@@ -0,0 +1,83 @@
+Testing the debug flags for dune-site
+
+  $ mkdir -p b c
+
+  $ for i in b; do
```

Comment:
can you inline this? (same below)

Reply:
I mean, instead of `for i in b; do ... done`, can you replace the `$i` by `b` in `...`?

Reply:
another option is to add `DUNE_SITE_DEBUG=1 dune exec ...` to an existing test.

Reply:
I'll change some existing tests then.